### PR TITLE
Json playcount

### PR DESCRIPTION
**Pull request by klucher**
Adding JSON Play Count functionality into Dev branch

### CODE REVIEW

File: BucStop/wwwroot/playcount.json
Comment (chrisseals98) [inline BucStop/wwwroot/playcount.json:1]: could you make the playcount 0 for the json? Just so it starts out at 0 when deployed.

File: .gitignore [hunk 1]
```diff
@@ -360,4 +360,5 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
+/BucStop/Models/Game.cs
```
Comment (chrisseals98) [inline .gitignore]: why was this added to the git ignore?

Comment (klucher) [reply .gitignore]: Not sure why Game.cs ended up in there. Removed

File: BucStop/wwwroot/playcount.json
Comment (klucher) [reply BucStop/wwwroot/playcount.json:1]: fixed

**Review by chrisseals98 — APPROVED** [review]
looks good, should work I think

**Review by Dylemperor — APPROVED** [review]
Chris said it should be good. I agree
